src/components/EditCoursePage/AdditionalMetadataFields.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+import RenderInputTextField from '../RenderInputTextField';
+import FieldLabel from '../FieldLabel';
+import RichEditor from '../RichEditor';
+
+function AdditionalMetadataFields(props) {
+  const {
+    disabled,
+  } = props;
+
+  return (
+    <div className="collapsible-card pgn_collapsible mt-4">
+      <div className="collapsible-body">
+        <FieldLabel text="Executive Education details" className="h3 font-weight-normal mb-3" />
+        <Field
+          name="additional_metadata.external_url"
+          component={RenderInputTextField}
+          label=<FieldLabel id="external_url.label" text="External landing page URL" />
+          disabled={disabled}
+        />
+        <Field
+          name="additional_metadata.external_identifier"
+          component={RenderInputTextField}
+          label=<FieldLabel id="external_identifier.label" text="2U system identifier" />
+          disabled={disabled}
+        />
+        <Field
+          name="additional_metadata.lead_capture_form_url"
+          component={RenderInputTextField}
+          label=<FieldLabel id="lead_capture_form_url.label" text="Lead capture form url" />
+          disabled={disabled}
+        />
+        <Field
+          name="additional_metadata.organic_url"
+          component={RenderInputTextField}
+          label=<FieldLabel id="organic_url.label" text="Organic URL" />
+          disabled={disabled}
+        />
+        <FieldLabel text="Stat 1" className="h3 font-weight-normal" />
```

Comment:
On another thought; since we are storing this field as Fact in DB, why not rename the header field Fact 1, Fact 2?